include/onnxruntime/core/session/onnxruntime_cxx_inline.h

```diff
@@ -753,6 +753,11 @@ inline void SessionOptionsImpl<T>::RegisterCustomOpsLibrary(const ORTCHAR_T* lib
   ThrowOnError(GetApi().RegisterCustomOpsLibrary_V2(this->p_, library_name));
 }
 
+template <typename T>
+inline void SessionOptionsImpl<T>::RegisterCustomOpsUsingFunction(const ORTCHAR_T* registration_function_name) {
+  ThrowOnError(GetApi().RegisterCustomOpsUsingFunction(this->p_, registration_function_name));
```

Comment:
>registration_function_name [](http://example.com/codeflow?start=65&length=26)

This should not compile, IMHO with ORTCHAR_T

Reply:
Yeah - I would have expected it to break as well.